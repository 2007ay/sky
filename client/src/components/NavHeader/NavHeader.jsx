import React from 'react';
import BpkSmallPriceIcon from 'bpk-component-icon/lg/price-alerts';

import STYLES from './NavHeader.scss';

const c = className => STYLES[className] || 'UNKNOWN';

const NavHeader = () => (
  <div className={c('NavHeader')}>
    <span> Filter</span>
    <span> Sort</span>
    <span>
      <BpkSmallPriceIcon className={c('NavHeader_price-alerts')} />
      <span>Price alerts</span>
    </span>
  </div>
);

export default NavHeader;
