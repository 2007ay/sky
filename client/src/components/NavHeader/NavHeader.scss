@import '~bpk-mixins';
@import '../base.scss';

.NavHeader {
  display: flex;
  padding: $bpk-spacing-sm 0;
  justify-content: space-between;
  align-items: center;
  background-color: $bpk-color-white;

  @include bpk-box-shadow-sm;

  &__hidden-text {
    @include bpk-visually-hidden;
  }

  span {
    color: $primary-txt-color;
    display: inline-block;
    padding: 0 $bpk-spacing-sm;
  }

  &_price-alerts {
    fill: $primary-txt-color;
    top: 5px;
    position: relative;
  }

}


@media screen and (min-width: 768px) {
  .NavHeader {
    padding: $bpk-spacing-sm 15%;
  }
}