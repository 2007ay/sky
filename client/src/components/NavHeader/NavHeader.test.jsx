import React from 'react';
import TestRenderer from 'react-test-renderer';

import NavHeader from './NavHeader';

describe('NavHeader', () => {
  it('should render correctly', () => {
    const tree = TestRenderer.create(<NavHeader />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
