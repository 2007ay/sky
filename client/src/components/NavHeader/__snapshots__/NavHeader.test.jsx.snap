// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavHeader should render correctly 1`] = `
<div
  className="NavHeader"
>
  <span>
     Filter
  </span>
  <span>
     Sort
  </span>
  <span>
    <svg
      className="NavHeader_price-alerts"
      height="24"
      style={
        Object {
          "height": "1.5rem",
          "width": "1.5rem",
        }
      }
      viewBox="0 0 24 24"
      width="24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M10 19.1h4.8c-.6 1.5-2.3 2.3-3.9 1.7-.8-.3-1.4-.9-1.7-1.7h.8zm8-8c0-2.5-1.6-4.8-4-5.7.2-1.1-.5-2.1-1.6-2.3-1.1-.2-2.1.5-2.3 1.6v.7C7.7 6.2 6 8.5 6 11v3l-2 3h16l-2-3v-2.9z"
      />
    </svg>
    <span>
      Price alerts
    </span>
  </span>
</div>
`;
